### PR TITLE
Renaming: GCP Function to GCP Monitor

### PR DESCRIPTION
**Pull request by joaquinfilipic-dynatrace**

### CODE REVIEW

File: gcp_iam_roles/dynatrace-gcp-monitor-metrics-role.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# For Kubernetes, Cloud Function counterpart is in setup.sh script
+# For Kubernetes. Cloud Function counterpart is in setup.sh script
```
Comment (piotrgorkowskidt) [inline gcp_iam_roles/dynatrace-gcp-monitor-metrics-role.yaml:1]:
```suggestion
# For Kubernetes, the Cloud Function counterpart is in setup.sh script
```

File: k8s/helm-chart/dynatrace-gcp-monitor/Chart.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@
 #     limitations under the License.
 
 apiVersion: v2
-name: dynatrace-gcp-function
-description: A Helm chart for dynatrace-gcp-function which provides the mechanism to pull all Google Cloud metrics and/or Google Cloud logs into Dynatrace.
+name: dynatrace-gcp-monitor
+description: A Helm chart for dynatrace-gcp-monitor which provides the mechanism to pull all Google Cloud metrics and/or Google Cloud logs into Dynatrace.
 type: application
 version: 0.3.0
```
Comment (piotrgorkowskidt) [inline k8s/helm-chart/dynatrace-gcp-monitor/Chart.yaml:19]: There is a warning "No newline at end of file"

```suggestion
version: 0.3.0

```

File: scripts/setup.sh [hunk 3]
```diff
@@ -261,7 +261,7 @@ elif [[ "$SERVING_APP_ENGINE" != "SERVING"  ]]; then
   exit
 fi
 
-debug "Select size of GCP Function"
+debug "Select size of Cloud Function"
```
Comment (piotrgorkowskidt) [inline scripts/setup.sh:264]: Nice catch 🥇

**Review by piotrgorkowskidt — DISMISSED** [review]

File: gcp_iam_roles/dynatrace-gcp-monitor-metrics-role.yaml (hunk 1 above)
Comment (joaquinfilipic-dynatrace) [reply gcp_iam_roles/dynatrace-gcp-monitor-metrics-role.yaml:1]: The . is deliberate. I can still add "The Cloud Function ...", though 😄

**Review by dtwwolinski — APPROVED** [review]

**Pull request merged**